### PR TITLE
fix accuracy calculation

### PR DESCRIPTION
**Pull request by KikuchiTomo**
Accuracy = (Σ_i w[i, i])/(Σ_i Σ_j w[i, j])

I think the formula is different.

After applying this modification, the accuracy rate is calculated to be the same as the following library

`from sklearn.metrics import accuracy_score`

### CODE REVIEW

**Comment by KikuchiTomo** [comment]
this pr is outdated.
